8Knot/queries/issues_query.py

```diff
@@ -76,6 +78,13 @@ def issues_query(self, repos):
     df["created"] = pd.to_datetime(df["created"], utc=True).dt.date
     df = df[df.created < dt.date.today()]
 
+    # reformat reporter_id and issue_closer
+    df["reporter_id"] = df["reporter_id"].astype(str)
+    df["reporter_id"] = df["reporter_id"].str[:15]
+
+    df["issue_closer"] = df["issue_closer"].astype(str)
+    df["issue_closer"] = df["issue_closer"].str[:15]
```

Comment:
query still broken on my end

Reply:
Have you rebuilt the container? I have on my side and it is working

Reply:
yeah works now